src/components/search/search-context.js

```diff
@@ -115,6 +115,21 @@ export const HelxSearch = ({ children }) => {
     return knowledgeGraphs.map(graph => graph._source.knowledge_graph.knowledge_graph)
   }
 
+  const fetchStudyVariable = async (_id, _query) => {
+    const studyVariables = await axios.post(`${helxSearchUrl}/search_var`, {
+      concept: _id,
+      index: 'variables_index',
+      query: _query,
+      size: 1000
+    }).then(response => {
+      return response.data.result.hits.hits
+    }).catch(error => {
+      console.error(error)
```

Comment:
Would like to have a place to route these one day.

Reply:
Yeah, I think my next update will add more notification UI when user interaction happens. That could go to the error notification :)